src/openassetio-core/include/openassetio/managerApi/ManagerInterface.hpp

```diff
@@ -938,6 +937,77 @@ class OPENASSETIO_CORE_EXPORT ManagerInterface {
                             const ExistsSuccessCallback& successCallback,
                             const BatchElementErrorCallback& errorCallback);
 
+  /**
+   * Callback signature used for a successful entity trait set query.
+   */
+  using EntityTraitsSuccessCallback = std::function<void(std::size_t, trait::TraitSet)>;
+
+  /**
+   * Provides the host with the @ref trait_set of one or more @ref
+   * entity "entities".
+   *
+   * For example, a host may use this to validate that a user-provided
+   * entity reference is appropriate for an operation.
```

Comment:
[Consider][minor] "For example" this early on soft-implies to my mind that this is _the_ use case for this.

```suggestion
   * An example use case of this could be a host validating that a user-provided
   * entity reference is appropriate for an operation.
```

Reply:
fwiw, I liked it.

Reply:
I'm struggling to improve on what's there, and since this is only a minor consideration, and Tom likes it, I'm going to leave as-is.